pkg/router/mine-service.go

```diff
@@ -17,6 +17,7 @@ func MineServiceUpload(r *gin.Engine, validate *validator.Validate, ApiVersion s
 		authUrl.POST("/mine-service/upload", mineservice.MineImageUpload)
 		authUrl.POST("/mine-service/url", mineservice.MineImageUrl)
 		authUrl.GET("/mine-service/get-all", mineservice.GetMinedImages)
+		authUrl.POST("/mine-service/upload-csv", mineservice.ParseUrlFromCsv)
```

Comment:
@noornee @usmansadik Does this task need to be exposed?

Reply:
no it doesn't, I'm gonna make changes to my code and revert the commit i made for this!

Reply:
@Emmrys-Jay, reverted the commit already